locales/pt-PT/translation.json

```diff
@@ -0,0 +1,281 @@
+  "Useful for miners or DeFi use.": "Útil para os mineradores ou uso DeFi.",
+  "More to come...": "Mais por vir...",
+  "You have great ideas you want to share?": "Tem boas ideias que quer partilhar?",
+  "Tell us!": "Diga-nos!",
+  "New address": "Novo endereço",
+  "Refresh": "Atualizar",
+  "Refresh data": "Atualizar dados",
+  "Alephium Logo": "Logo Alephium",
+  "Alephium": "Alephium",
+  "WALLET": "CARTEIRA",
+  "CURRENT WALLET": "CARTEIRA ATUAL",
+  "MENU": "MENU",
+  "Overview": "Visão geral",
+  "Lock": "Bloquear",
+  "Enter password": "Introduza a palavra-chave",
+  "Login": "Iniciar sessão",
+  "Security Check": "Verificação de segurança",
+  "Alright! Time to check if you got your words right!": "Boa! Está na hora de verificar se as suas palavras estão correctas!",
+  "Select the words in the right order. Ready?": "Selecione as palavras na ordem certa. Pronto?",
+  "Ready!": "Pronto!",
+  "Something went wrong when creating encrypted wallet.": "Algo correu mal ao criar a carteira encriptada.",
+  "Select the words in the right order.": "Selecione as palavras na ordem certa.",
+  "It seems like you made a mistake in the words' order. But don't worry, you can reorder the words by dragging them around.": "Parece que cometeu um erro na ordem das palavras. Mas não se preocupe, pode reordenar as palavras arrastando-as de um lado para o outro.",
+  "Great job! Remember to keep those words safe.": "Excelente trabalho! Lembre-se de manter essas palavras em segurança.",
+  "Continue": "Continuar",
+  "Password is too weak": "A palavra-passe é demasiado fraca",
+  "Insecure password": "Palavra-passe insegura",
+  "Wallet name is too short": "O nome da carteira é demasiado curto",
+  "Wallet name already taken": "Nome da carteira ocupado",
+  "New wallet name": "Novo nome da carteira",
+  "New password": "Nova palavra-passe",
+  "Retype password": "Reescreva a palavra-passe",
+  "Passwords are different": "As palavras-passe são diferentes",
+  "Make sure to keep your password secured as it cannot be changed in the future.": "Assegure-se de manter a sua palavra-passe segura, pois não pode ser alterada no futuro.",
+  "Alephium doesn't have access to your wallet.\nYou are the only owner.": "A Alephium não tem acesso à sua carteira.\nÉ o único proprietário.",
+  "Back": "Voltar",
+  "Type your recovery phrase": "Escreva a sua frase de recuperação",
+  "Error while importing wallet": "Erro ao importar a carteira",
+  "Make sure to store the words in a secure location! They are your wallet's secret recovery phrase.": "Assegure-se de guardar as palavras num local seguro! São a frase secreta de recuperação da sua carteira.",
+  "All good? Let's continue!": "Tudo bem? Vamos continuar!",
+  "Everything is ready!": "Tudo está pronto!",
+  "Welcome to Alephium.": "Bem-vindo a Alephium.",
+  "Let's go!": "Vamos lá!",
+  "welcomeScreenOneAddressPerGroupMessage": "Utilizador avançado: pretende começar com <1>one address per group for mining or DeFi?</1>",
```

Comment:
@JKtranslator there's a missing translation here:

`one address per group for mining or DeFi`. Could you help, please? :)